src/Flex.php

```diff
@@ -1010,14 +965,13 @@ public static function getSubscribedEvents(): array
             ScriptEvents::POST_INSTALL_CMD => 'install',
             ScriptEvents::PRE_UPDATE_CMD => 'configureInstaller',
             ScriptEvents::POST_UPDATE_CMD => 'update',
-            ScriptEvents::POST_AUTOLOAD_DUMP => 'updateAutoloadFile',
-            'auto-scripts' => 'executeAutoScripts',
```

Comment:
this change looks wrong to me. It will probably break things like `composer run auto-scripts`